Remove invocations of curl. Instead, we will provide the sources in the ebuild.

diff --git a/Makefile.orig b/Makefile
index 97c6500..65f7e83 100644
--- a/Makefile.orig
+++ b/Makefile
@@ -185,12 +185,6 @@ RUSTCSRC: $(RUSTC_SRC_DL)
 #
 # NIGHTLY:
 ifeq ($(RUSTC_SRC_TY),nightly)
-rustc-nightly-src.tar.gz: $(RUSTC_SRC_DES)
-	@export DL_RUST_DATE=$$(cat rust-nightly-date); \
-	export DISK_RUST_DATE=$$([ -f $(RUSTC_SRC_DL) ] && cat $(RUSTC_SRC_DL)); \
-	echo "Rust version on disk is '$${DISK_RUST_DATE}'. Downloading $${DL_RUST_DATE}."; \
-	rm -f rustc-nightly-src.tar.gz; \
-	curl -sS https://static.rust-lang.org/dist/$${DL_RUST_DATE}/rustc-nightly-src.tar.gz -o rustc-nightly-src.tar.gz
 
 $(RUSTC_SRC_DL): rust-nightly-date rustc-nightly-src.tar.gz rustc-nightly-src.patch
 	@export DL_RUST_DATE=$$(cat rust-nightly-date); \
@@ -204,10 +198,6 @@ $(RUSTC_SRC_DL): rust-nightly-date rustc-nightly-src.tar.gz rustc-nightly-src.pa
 else
 # NAMED (Stable or beta)
 RUSTC_SRC_TARBALL := rustc-$(RUSTC_VERSION)-src.tar.gz
-$(RUSTC_SRC_TARBALL): $(RUSTC_SRC_DES)
-	@echo [CURL] $@
-	@rm -f $@
-	@curl -sS https://static.rust-lang.org/dist/$@ -o $@
 $(RUSTC_SRC_DL): $(RUSTC_SRC_TARBALL) rustc-$(RUSTC_VERSION)-src.patch
 	tar -xf $(RUSTC_SRC_TARBALL)
 	cd $(RUSTCSRC) && patch -p0 < ../rustc-$(RUSTC_VERSION)-src.patch;
